### DISCUSSION TITLE
Dataset request

### DISCUSSION THREAD

**Discussion by Silintone** [Mr-MIBonk/M.I.B._More-Incredible-Bash#156] [Q&A]
@harman-f .

Would it be possible , like with the subwoofer gain script, to be able to do the same thing but for my 2 rear doors, to add  +2db or better +4db. ( PS I'm enjoying the +8db sub :) so thanks for that).
Its killing me to have to install a amp just to increase rear volume. 
My car has a issue with the rear door been half the volume of the front doors, upgrade speakers don't work, ive tried a pricey set and a cheap set, no luck there as volume remains much lower then the front.
Fading the sound to the rear to balance it out , deprives the car of total volume been equal to about half volume even at max.
I can upload data set if you'd be able to assist.
Thanks in advance.

**Comment by harman-f** [comment]
This is way to sepcial request ;)

I'm also not awar, that someone decoded this part of the sound datasets. This means editing the soundprofile of the car for a specific speaker setup.

Did you retrofit the rear speakers?

**Comment by Silintone** [comment]
haha i figured i was maybe pushing my luck. :P 

well maybe this helps your projects ;) ( i have the scripts thats used to decode the datasets using  010 editor from the author of that page) 
https://mqb-blog.com/en/2022/02/21/sound-dataset/#comment-6109465535

I got speakers that cost about 95 euro from a local audio specialist , installed them with custom made spacers, sound was softer.
Returned them and got a cheap set of around 45 euro thinking the good ones was to much for the internal amp, but the cheap ones resulted in the same result , softer then the factory speakers........ painful experience really.. so returned those also.

So the speakers of the car is still factory, i dont want to do the front doors because then there will be even less sound in the rear .
Getting a MHI2Q or amping the speakers is the next step but i want to avoid that route.

**Comment by harman-f** [comment]
Get factory speaker for the rear and the proper dataset. All should be better 😉

**Reply by Silintone** [reply]
haha i have factory speakers in the rear , and thats why i asked if you would be able to XD

**Reply by harman-f** [reply]
Have you changed your dataset on unit to the one with stock rear speakers?

**Comment by Silintone** [comment]
Only  changes in my dataset is the ones we tested with for the sub, further changes in the dataset has not been done.

**Reply by harman-f** [reply]
I‘m not talking about changes. 
there is a specific dataset for each speaker configuration. If you change your speaker config you also have switch to that dataset!
Your stock dataset does not include the right settings for the rear speakers. Hence, they sound like ****

**Comment by Silintone** [comment]
Ahhhhh im confused now , i tested with upgrade speakers( not exactly the same speakers Bruce_Miranda sells , but similar - people praise that upgrade , they dont change any datasets)
 But i have the factory speakers in and havent changed/uploaded datasets for anything other then the sub,  so i understand that the rear speaker dataset is factory ( should be correct dataset then) 

Would it be possible to point me in the right direct to get different dataset for the rear ? or how to correct them ?

**Reply by harman-f** [reply]
maybe I'm wrong, let me check with someone ;)

**Reply by harman-f** [reply]
what was your car model again?

**Comment by Silintone** [comment]
No probs, and thank you.

2016 A4 B9 sedan  /  A.S.S  (Centre + sub ) /   MHI2 mmi ,

**Reply by harman-f** [reply]
MHI2 is stock or retrofit?

**Comment by Silintone** [comment]
Stock , 

 MHI2Q would be my last resort to retrofit if i cant fix audio levels/quality

**Reply by harman-f** [reply]
can you please share your dataset.csv from M.I.B backup again here.

**Comment by Silintone** [comment]
Yea sure , here we go , did a fresh mib download , and backup :) 

[MU1438-datasets.csv](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11056092/MU1438-datasets.csv)

**Reply by harman-f** [reply]
A fresh one…

go into GEM to backup/restore and restore backup from unit. 

You will find a new folder in /backup/ get me the dataset.cab from that folder.

**Comment by Silintone** [comment]
[MU1438-datasets.csv](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11059392/MU1438-datasets.csv)


Hope this is what your looking for , got it from the NAND-Backup

**Reply by Silintone** [reply]
Please let me know if this is the correct one from the NAND backup folder

**Comment by elysion0619** [comment]
My 2017 A4 also has the same problem. The volume of the speaker on the rear door panel is significantly lower, affecting the stereo listening experience. I hope there can be a solution. I also did not find dataset.cab in /backup/, only MU0729-datasets.csv.

**Comment by Silintone** [comment]
@elysion0619 , let me know if you find a solution, i shall do the same.
Not to sure if Harman can assist us with this.

**Comment by AmarAtcha** [comment]
Same for my a4 b9 2019 with B&O
The volume of the speakers on the rear is LOWER
